web/src/settings_components.ts

```diff
@@ -2006,7 +2006,9 @@ export function get_group_assigned_user_group_permissions(group: UserGroup): {
 
 export function set_up_folder_dropdown_widget(sub?: StreamSubscription): DropdownWidget {
     const folder_options = (): dropdown_widget.Option[] => {
-        const folders = channel_folders.get_channel_folders();
+        const folders = channel_folders
+            .get_channel_folders()
```

Comment:
List returned by `get_channel_folders` is sorted by ID, not sure if we need that. `get_channel_folders` function is used only here currently.